.babelrc

```diff
@@ -3,5 +3,5 @@
         // Don't transpile modules since Rollup does that
         [ "env", { modules: false } ]
     ],
-    "plugins": [ "external-helpers" ]
+    "plugins": [ "external-helpers", "babel-plugin-transform-object-rest-spread" ]
```

Comment:
Does the Fuse library use these ES features internally? Just want to know why this is required.

Reply:
Yes [`Fuse.js` uses the spread operator](https://github.com/krisk/Fuse/blob/master/dist/fuse.esm.js#L215).